src/gui/mzroll/mainwindow.cpp

```diff
@@ -2143,6 +2145,38 @@ void MainWindow::_warnIfNISTPolarityMismatch()
     }
 }
 
+void MainWindow::_confirmMsMsType()
+{
+    QList<mzSample*> msMsSamples;
+    for (auto sample : samples) {
```

Comment:
@saiful-elucidata, this function makes use of all the samples present in the session and not the newly loaded. 

So, in case we load the PRM dataset in the El-MAVEN session, we get the prompt which is expected behavior. After loading the PRM dataset, if we try to load the LCMS dataset in the same session we again get the prompt, which in my viewpoint must not appear. 

But it is up to you if we would like to handle it or not.

Reply:
To be able to remember which type of dataset(s) have been loaded into the software we will need an additional settings/MavenParameters storage. I would like to avoid that for something that users are very unlikely to do. The only time you would load samples in batches in El-MAVEN would be when it's too large to be loaded at once (because of memory issues) and even then the batches of sample files would be from the same dataset. Loading different types of datasets together has never been handled or allowed within the software. This is fine in my opinion.

Reply:
This makes sense. I will approve it. :)